### PR TITLE
Addressed (most of) Issue #531

### PR DESCRIPTION
**Pull request by RemingtonGerras**
## Description

- UVI is no longer off by a factor of 10 (divided by 10)
- UVI overflow issue is fixed by adding 2**7 to negative (overflowed) values
- Changed temperature from Fahrenheit to Celsius

## Motivation and Context

[Issue #531](https://github.com/simonsobs/socs/issues/531)

## How Has This Been Tested?

I ran python in terminal and created functions that take dummy input values and output the changed value. 
These functions reflected the changes I made in the agent.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

There are comments in code that should be changed (instances of 'fahrenheit' to 'celsius')

### CODE REVIEW

File: socs/agents/vantagepro2/drivers.py [hunk 1]
```diff
@@ -312,6 +312,13 @@ def receive_data(self):
         loop_data['time_sunrise'] = byte_data[75]
         loop_data['time_sunset'] = byte_data[76]
 
+        # Correct UV Index by a factor of 10 and fix overflow
+        uvi = loop_data['UV']
+        if uvi < 0:
+            uvi += 2**7
```
Comment (mhasself) [inline socs/agents/vantagepro2/drivers.py:318]: Great to see ... this should be 2**8 = 256, right?  i.e., -127 becomes +129.

Comment (RemingtonGerras) [reply socs/agents/vantagepro2/drivers.py:318]: oops yes that makes sense

File: socs/agents/vantagepro2/drivers.py [hunk 2]
```diff
@@ -241,27 +241,27 @@ def receive_data(self):
         loop_data['packet_type'] = byte_data[4]
         loop_data['next_record'] = byte_data[5]
         loop_data['barometer'] = byte_data[6] / 1000.0
-        loop_data['temp_inside'] = byte_data[7] / 10.0
+        loop_data['temp_inside'] = (byte_data[7] / 10.0 - 32) * (5 / 9)
```
Comment (mhasself) [inline socs/agents/vantagepro2/drivers.py:244]: Please write a function that converts F to C, and use it on all of these.  This is like the classic example of when should use a function.

**Review by mhasself — CHANGES_REQUESTED** [review]
Thanks, UV fix looks good.  Please use function for F_to_C.  It is then self-documenting and less prone to little typos.

File: socs/agents/vantagepro2/drivers.py (hunk 2 above)
Comment (RemingtonGerras) [reply socs/agents/vantagepro2/drivers.py]: you're right :(

**Review by mhasself — APPROVED** [review]

**Pull request merged**